src/utils/Cypher.ts

```diff
@@ -10,6 +15,7 @@ const MODULE_NAME = 'Cypher';
 
 export default class Cypher {
   private static configuration: CryptoConfiguration;
+  private static cryptoKey: CryptoKeySetting;
```

Comment:
In the end, only that attribute will remain with a more sensible name like `cryptoSetting`, the key in the configuration file will be used to initially populate all tenants crypto settings with a migration.